src/client/src/ui/spotTile/spotTileOperations.ts

```diff
@@ -165,13 +172,14 @@ export const spotTileReducer = (state: any = {}, {type, payload}) => {
     case ACTION_TYPES.DISMISS_NOTIFICATION:
       return updateSpotTile(state, payload.symbol, { notification: null })
     case ACTION_TYPES.PRICING_STALE:
-      const stalePrice = _.pick(state, payload.symbol)
-      if (stalePrice) {
-        stalePrice[payload.symbol].priceStale = true
-        stalePrice[payload.symbol].notification = buildNotification({}, stalePriceErrorMessage)
```

Comment:
😱

Reply:
Doesn't feel right that notification right there, but I leave it for now.